bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -37,9 +37,55 @@
 				<description>Seconds between fetching background updates about the air purifiers / humidifiers.</description>
 				<default>60</default>
 			</parameter>
+			<parameter name="outletPollInterval" type="integer" min="5" step="1" unit="s">
+				<label>Oultet Poll Rate</label>
+				<description>Seconds between fetching background updates about the outlets.</description>
+				<default>60</default>
+			</parameter>
 		</config-description>
 	</bridge-type>
 
+	<thing-type id="outlet">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Outlet via VeSync</label>
+		<description>An Outlet uplinking to VeSync</description>
+
+		<channels>
+			<channel id="enabled" typeId="deviceEnabledType"/>
+			<channel id="current" typeId="currentType"/>
+			<channel id="energy" typeId="energyType"/>
+			<channel id="power" typeId="powerType"/>
+			<channel id="voltage" typeId="voltageType"/>
+			<channel id="highestVoltage" typeId="highestVoltageType"/>
+			<channel id="voltagePTStatus" typeId="voltagePTStatusType"/>
+			<channel id="energyWeek" typeId="energyWeekType"/>
+			<channel id="energyMonth" typeId="energyMonthType"/>
+			<channel id="energyYear" typeId="energyYearType"/>
+		</channels>
+
+		<properties>
+			<property name="Device Name"/>
+			<property name="Device Type"/>
+			<property name="MAC Id"/>
+			<property name="Device Family"/>
+		</properties>
+		<representation-property>macId</representation-property>
+
+		<config-description>
+			<parameter name="macId" type="text">
+				<label>MAC Id</label>
```

Comment:
`MAC Id`,  I think this should be `MAC Address` ?

Reply:
Fixed in 7893d2e